django/contrib/sphinx/__init__.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+Sphinx extension for django projects.
+
+Usage::
+    Add ``django.contrib.sphinx`` to the ``extensions`` list in your sphinx config.
+"""
+import inspect
+
+from django.db.models.fields.files import FileDescriptor
+
+# Monkey patch Fix Django's FileFields
+FileDescriptor.__get__ = lambda self, *args, **kwargs: self
```

Comment:
My understanding is that this shouldn't be necessary as of 9f6b704769ba5bc0daafc25340d3dc28b18d8fb1.